Flutter/flutter_gallery_app/lib/src/presentation/text_field/screen/text_field_page.dart

```diff
@@ -0,0 +1,169 @@
+import 'package:auto_route/auto_route.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_gallery_app/src/presentation/container/widgets/view_option_swich.dart';
+import 'package:flutter_gallery_app/src/presentation/text_field/bloc/text_field_bloc.dart';
+import 'package:flutter_gallery_app/src/presentation/text_field/bloc/text_field_event.dart';
+import 'package:flutter_gallery_app/src/presentation/text_field/bloc/text_field_state.dart';
+
+@RoutePage()
+class TextFieldPage extends StatelessWidget {
+  const TextFieldPage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocProvider<TextFieldBloc>(
+      create: (context) => TextFieldBloc(),
+      child: const TextFieldView(),
+    );
+  }
+}
+
+class TextFieldView extends StatefulWidget {
```

Comment:
You should separate the 'TextFieldView' class into a separate file. And then change name "TextFieldView" to "TextFieldWidget"

Reply:
Thank you, I have resolved the problem in new commit